packages/mui-base/src/Popover/Provider/PopoverProvider.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { FloatingDelayGroup } from '@floating-ui/react';
+import type { PopoverProviderProps } from './PopoverProvider.types';
+
+/**
+ * Provides a shared delay for popovers so that once a popover is shown, the rest of the popovers in
+ * the group will not wait for the delay before showing.
+ *
+ * Demos:
+ *
+ * - [Popover](https://mui.com/base-ui/react-popover/)
+ *
+ * API:
+ *
+ * - [PopoverGroup API](https://mui.com/base-ui/react-popover/components-api/#popover-group)
+ */
+function PopoverProvider(props: PopoverProviderProps) {
```

Comment:
We decided that the provider is needed only for tooltips (https://www.notion.so/mui-org/base-ui-Weekly-Meeting-2024-06-03-e1afec310a444221afda2b1809bfdf04?pvs=4#29b2aa96c2e147b683fae32e6407838e)